fili-core/src/test/groovy/com/yahoo/bard/webservice/application/LoadTaskSpec.groovy

```diff
@@ -231,4 +237,42 @@ class LoadTaskSpec extends Specification {
         cleanup:
         scheduler.shutdownNow()
     }
+
+    class FailingTestLoadTask extends LoadTaskSpec.OneOffTestLoadTask {
+
+
+        public static final String UNIQUE_VALUE_1 = "UNIQUE_VALUE_1"
+
+        FailingTestLoadTask(long delay) {
+            super(delay)
+        }
+
+        @Override
+        void runInner() {
+            throw new TimeoutException(UNIQUE_VALUE_1);
+        }
+    }
+
+
+    def "Test error gets logged on failed test"() {
+        setup:
+        setup: "Instantiate a task scheduler"
+        ObjectMapper MAPPER = new ObjectMappersSuite().getMapper()
+
+
+        TaskScheduler scheduler = new TaskScheduler(2)
+        LoadTask<Boolean> loader = new FailingTestLoadTask(expectedDelay)
+        loader.setFuture(
+                scheduler.schedule(
+                        loader,
+                        1,
+                        TimeUnit.MILLISECONDS
+                )
+        )
+        Thread.sleep(10)
+
+        expect:
+        loader.failed
+        logAppender.getMessages().find { it.contains(/UNIQUE_VALUE_1/) }
```

Comment:
Change `"UNIQUE_VALUE_1"` to `FailingTestLoadTask.UNIQUE_VALUE_1` for better maintainability?